packages/web/public/static/locales/it/profile.json

```diff
@@ -1,36 +1,40 @@
 {
   "card": {
-    "speaks": "speaks",
-    "learns": "learns",
-    "likes": "likes"
+    "speaks": "parla",
+    "learns": "studia",
+    "likes": "interessi" // oh damn, the literal translation "piace" this is intransitive, so I replaced this with ("interessi") "interests"
   },
   "stats": {
     "summary": {
-      "title": "Summary",
-      "userAge": "{{name}} has been journaling on Journaly since {{joinDate}} (that's {{age}}).",
-      "postCount": "In that time they've published {{postCount}} posts in {{languageCount}} different languages.",
-      "noPosts": "So far they haven't written any posts.",
-      "commentCounts": "They've also left {{postCommentCount}} general comments on posts and {{threadCommentCount}} inline feedback comments, helping other users improve their language skills.",
-      "noComments": "They haven't left any feedback on other journalers' posts yet.",
-      "thanksCount": "For their hard work helping others, they've been thanked {{thanksCount}} times."
+      "title": "In breve",
+      "userAge": "{{name}} è utente di Journaly dal {{joinDate}} ({{age}} anni).", // is {{age}} a number, and if so is it actually expressed in years?
```

Comment:
It might be helpful to see it in the interface, which you can see on any profile in the Stats tab, such as https://journaly.com/user/harisont and then click on "Stats" to switch to your Activity Stats. We've improved how we use i18n with dates and relative time words, etc. now but still have quite a few places where we need to refactor things. For now you don't need "anni" here and unfortunately the age string will be displayed in English, but that will be fixed when we can complete that refactor 😊